mne/time_frequency/psd.py

```diff
@@ -120,13 +129,14 @@ def compute_epochs_psd(epochs, picks=None, fmin=0, fmax=np.inf, n_fft=256,
         Max frequency of interest
     n_fft : int
         The length of the tapers ie. the windows. The smaller
-        it is the smoother are the PSDs.
-    pad_to : int | None
-        The number of points to which the data segment is padded when
-        performing the FFT. If None, pad_to equals `n_fft`.
+        it is the smoother are the PSDs. The default value is 256.
+        If ``n_fft > len(epochs.times)``, it will be adjusted down to
+        ``len(epochs.times)``.
     n_overlap : int
-        The number of points of overlap between blocks. The default value
-        is 0 (no overlap).
+        The number of points of overlap between blocks. Will be adjusted
+        to be <= n_fft.
+    proj : bool
+        Apply SSP projection vectors.
```

Comment:
Personally I prefer *not* adding the defaults. I find it more DRY (change the default you only need to remember to change it in one place), and introspecting editors and doc builders are smart enough to know to show the params. Have we settled on adding defaults in mne-python? The numpy doc says that defaults can be added, but it doesn't make it sound mandatory. I'm okay with doing it if numpy/scipy/sklearn seem to be headed in that direction, though.